docs/applications/add-application-deploy-status-to-commit.md

```diff
@@ -0,0 +1,141 @@
+- Insomnia: [Import and Export Data: Import Data](https://docs.insomnia.rest/insomnia/import-export-data#import-data)
+
+NOTE: Insomnia uses curly braces syntax for environment variables, to avoid errors you need to disable the nunjuncks template feature for the request body. Further instructions at [Insomnia's FAQ](https://docs.insomnia.rest/insomnia/faq#how-can-i-temporarily-disable-nunjucks-template)
+
+1. Take note of the `Application ID` (`<application_id>`) and `Deploy Pipeline ID` (`<pipeline_id>`) from the application which you want to watch the action statuses
+   - Follow the [Deploy Application using an Incoming Hook](/docs/applications/deploy-application-using-incoming-hook) guide for detailed instructions
+
+1. Take note of the `Target URL` (`<target_url>`), `Request Headers` (`<request_headers>`) and `Request Body` (`<request_body>`) fields according to the source provider where the application' source code is hosted, by following the source provider's instructions on the links bellow:
+   - Bitbucket: [REST APIs: Create a build status for a commit](https://developer.atlassian.com/cloud/bitbucket/rest/api-group-commit-statuses/#api-repositories-workspace-repo-slug-commit-commit-statuses-build-post)
+   - Github: [REST API: Create a commit status](https://docs.github.com/en/rest/commits/statuses#create-a-commit-status)
+   - Gitlab: [REST API: Set the pipeline status of a commit](https://docs.gitlab.com/ee/api/commits.html#set-the-pipeline-status-of-a-commit)
+
+1. On your local machine, in a terminal window, submit a request to Devopness API endpoint `POST /hooks/outgoing` to create a outgoing for the `action.started` event, replacing `<application_id>`, `<pipeline_id>`, `<target_url>`, `<request_headers>` and `<request_body>`
+
+   - For further instructions, follow the guide [Create an Outgoing Webhook](/docs/webhooks/create-outgoing-webhook)
+
+   {% raw %}
+   ```bash
+   curl --request POST \
+     --url https://api.devopness.com/hooks/outgoing \
+     --header 'Accept: application/json' \
+     --header 'Authorization: Bearer <access_token>' \
+     --header 'Content-Type: application/json' \
+     --data '{
+       "name": "GitHub CI(build)",
+       "action_type": "deploy",
+       "resource_type": "application",
+       "resource_id": <application_id>,
+       "target_url": "https:\/\/<target_url>\/{{ action.triggered_from.hook_parsed_variables.commit_hash }}",
+       "settings": {
+         "request_headers": [
+           {
+             "name": "Authorization",
+             "value": "Bearer {{ application.source_provider.access_token }}"
+           },
+           // NOTE: add Request Headers (`<request_headers>`) here
+         ],
+         "request_body": {
+           // NOTE: review the fields bellow according to Request Body (`<request_body>`)
+           "state": <"INPROGRESS" | "pending" | "running">,
```

Comment:
@thlmenezes I believe that you should not add examples here, I understand that you tried to put the respective 'pending' status of each source provider, but this can cause confusion, for example, the person thinks that he can use any of these for any source provider, so I believe make it better like this:

```suggestion
         "request_body": {
           // NOTE: review the fields bellow according to Request Body of your source provider (`<request_body>`)
           "state": <source_provider_pipeline_status>,
```

Reply:
Post updated